.github/workflows/ci.yml

```diff
@@ -167,6 +167,27 @@ jobs:
       - name: Run tests
         run: dart run test -j 2 -t node -r expanded
 
+  browser-test:
+    name: "Browser Tests | Dart ${{ matrix.dart-channel }}"
+
+    strategy:
+      matrix:
+        dart-channel: [stable]
+      fail-fast: false
+
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: browser-actions/setup-chrome@v1
+    - uses: dart-lang/setup-dart@v1.2
+      with:
+        sdk: ${{ matrix.dart-channel }}
+    - run: dart pub get
+    - run: dart run grinder before-test
+    - run: google-chrome --version
+    - run: dart run test --platform chrome -j 2 -t browser -r expanded
```

Comment:
For some reason [all tests are running in Chrome](https://github.com/sass/dart-sass/actions/runs/4444257847/jobs/7802292764?pr=1895), not only those tagged with `browser`. Am I missing something?

Reply:
Don't use tags, use @TestOn and separate vm from browsers tests

Reply:
Fixed in https://github.com/oddbird/dart-sass/commit/e238290c1738e761d3ebba5ae40016a754fa5268